src/core/interfaces.ts

```diff
@@ -451,14 +451,26 @@ export namespace CoreChartProps {
   }
   export interface TooltipPointProps {
     item: TooltipContentItem;
+    /**
```

Comment:
nit: I would remove api-docs from here as we don't generate documentation for core and the property purpose already looks clear